firstvoices/backend/serializers/dictionary_serializers.py

```diff
@@ -206,3 +206,34 @@ class Meta:
                 "split_words_base",
             )
         )
+
+
+class WritableRelatedDictionaryEntrySerializer(serializers.PrimaryKeyRelatedField):
+    def use_pk_only_optimization(self):
+        return False
+
+    def to_representation(self, value):
+        return DictionaryEntrySummarySerializer(context=self.context).to_representation(
+            value
+        )
+
+
+class RelatedDictionaryEntrySerializerMixin(metaclass=serializers.SerializerMetaclass):
+    related_dictionary_entries = WritableRelatedDictionaryEntrySerializer(
+        required=False,
+        many=True,
+        queryset=dictionary.DictionaryEntry.objects.all(),
+    )
+
+    def validate(self, attrs):
```

Comment:
I added this override method to be equivalent to the `SameSite` validator. During testing I found that specified validators are not applied in many to many nested fields. Verified in the documentation here: https://www.django-rest-framework.org/api-guide/validators/#updating-nested-serializers